compiler-rt/test/rtsan/stack_suppressions.cpp

```diff
@@ -1,4 +1,5 @@
 // RUN: %clangxx -fsanitize=realtime %s -o %t
+// RUN: %env_rtsan_opts=halt_on_error=false %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-NOSUPPRESSIONS
```

Comment:
Added this check here to make sure that these would actually cause errors before suppressing them